client/src/featureform/resources.py

```diff
@@ -684,8 +697,15 @@ class Feature:
     description: str
     variant: str = "default"
     schedule: str = ""
+    status: str = ""
```

Comment:
Should the default value be `"NO_STATUS"`?

Reply:
yes